test/server/cards/01_SOR/ChewbaccaLoyalCompanion.spec.js

```diff
@@ -0,0 +1,23 @@
+describe('Chewbacca, Loyal Companion', function() {
+    integration(function() {
+        describe('Chewbacca\'s when attacked ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['wampa']
+                    },
+                    player2: {
+                        groundArena: [{ card: 'chewbacca#loyal-companion', exhausted: true }]
+                    }
+                });
+            });
+
+            it('readies Chewbacca when he is attacked', function () {
+                this.player1.clickCard(this.wampa);
```

Comment:
Can we set it up so that it readies twice in the same phase just to confirm the trigger works multiple times. Can either exhaust it manually and pass player 2 or have it do an attack, either's fine

Reply:
Sure.